dashboard/src/components/Footer.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <footer>
+    <p>
+      Coded with lots of <img src="https://discord.com/assets/25c09e6fde32411da2b0da00f5cb9c84.svg" alt="coffee_logo"> by 
+      <a href="https://npstr.space/">Napster</a>
+    </p>
+  </footer>
+</template>
+
+<style lang="scss">
+  footer {
+
+    font-size: 13px;
+
+    img {
+      height: 13px;
```

Comment:
The other parts of the web frontend codebase is currently sticking to `em` units. If there a reason in this specific case to use px, let's document it.